sdk/cost-management/arm-costmanagement/package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "@azure/arm-costmanagement",
+  "author": "Microsoft Corporation",
+  "description": "CostManagementClient Library with typescript type definitions for node.js and browser.",
+  "version": "1.0.0",
```

Comment:
Is this supposed to be a 1.0.0 GA?

Reply:
This is an existing stable services. Maybe we could discuss it in the api version meeting as well ?